src/govuk/components/summary-list/template.test.js

```diff
@@ -9,461 +9,225 @@ const { render, getExamples } = require('../../../../lib/jest-helpers')
 
 const examples = getExamples('summary-list')
 
-describe('Data list', () => {
-  it('default example passes accessibility tests', async () => {
-    const $ = render('summary-list', examples.default)
-
-    const results = await axe($.html(), {
-      rules: {
-        // In newer versions of the HTML specification wrapper
-        // <div>s are allowed in a definition list
-        dlitem: { enabled: false },
-        'definition-list': { enabled: false }
-      }
-    })
-    expect(results).toHaveNoViolations()
-  })
-  it('renders classes', async () => {
-    const $ = render('summary-list', {
-      classes: 'app-custom-class'
+describe('Summary list', () => {
+  describe('by default', () => {
```

Comment:
@hannalaakso I think I did this because "default example" implies the tests are only using `examples.default` to test against and more often than not they were using other custom example(s). Perhaps we need to look more widely at what we actually include in the default example, but I tried to avoid changing the existing examples too much in this piece of work